packages/react-reconciler/src/ReactFiberCommitViewTransitions.js

```diff
@@ -590,61 +586,6 @@ export function restoreNestedViewTransitions(changedParent: Fiber): void {
   }
 }
 
-export function cancelViewTransitionHostInstances(
```

Comment:
This helper was only used when layout was "none" but update was not. If both were "none" we don't apply the names in the first place. Which is what happens if just update is "none" now and this case is not expressible.